src/main/java/wolox/training/models/Book.java

```diff
@@ -99,6 +117,8 @@ public String getYear() {
   }
 
   public void setYear(String year) {
```

Comment:
In order to make it even better, you could check that the `year` is not greater than the actual.

Reply:
done!